cmd/prefetchfiles-nri-plugin/prefetchConfig.toml

```diff
@@ -0,0 +1,3 @@
+[file_prefetch]
```

Comment:
We should not save the example NRI plugin config file in the `cmd` folder, how about moving it to `misc/nri-prefetch`

Reply:
fixed!